### PR TITLE
feature(navbar): Display enabled, approved count next to org #342

### PR DESCRIPTION
**Pull request by blaino**
This feature adds a slice of what is requested in #342, namely the
organization counts. These can be found as badges next to organizations
in the Orgranizations pull down. The data comes from calling an existing
grails backend endpoint (which had to be modified slightly to allow
users - not just org stewards and admin - to access the endpoint). We
also had to fork the react-select-box repo to allow adding tags with
count information to options.

In terms of React/Reflux, the Organizations component listens to a new
store, CountsStore which in turn listens to a new action,
fetchCounts. fetchCounts calls the ListingApi with the enabled and
approved query parameters set.

### CODE REVIEW

**Comment by blaino** [comment]
@wski, this one is a little more substantive. Please have a look when you get a chance.

File: app/js/webapi/Listing.js [hunk 1]
```diff
@@ -196,6 +196,14 @@ var ListingApi = {
         });
     },
 
+    getCounts: function (options) {
+        var url = API_URL + '/api/listing?' + $.param(options);
```
Comment (wski) [inline app/js/webapi/Listing.js]: > var url = `${ API_URL }/api/listing?${ $.param(options) }`;
is sexier, change if you want.

File: package.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
     "react-input-placeholder": "1.0.0",
     "react-radio-group": "1.0.0",
     "react-router": "0.12.4",
-    "react-select-box": "1.0.0",
+    "react-select-box": "https://github.com/ozone-development/react-select-box.git#master",
```
Comment (wski) [inline package.json]: > git://github.com/ozone-development/react-select-box.git#master

**Comment by wski** [comment]
:+1:

**Comment by wski** [comment]
re :+1:

**Comment by blaino** [comment]
@clarkjefcoat Can I get you to take a look at this? (Goes with https://github.com/ozone-development/ozp-rest/pull/96).

**Comment by clarkjefcoat** [comment]
Tested in conjunction with ozone-development/ozp-rest#96
Loaded center in a browser.  In different browser I deleted a listing.  Then in my original browser I switched to HUD and then back to center.  The counts were updated correctly.
:+1:

**Pull request merged**